src/main/resources/l10n/JabRef_de.properties

```diff
@@ -127,6 +127,10 @@ Cancel=Abbrechen
 
 Cannot\ add\ entries\ to\ group\ without\ generating\ keys.\ Generate\ keys\ now?=Einträge können einer Gruppe nicht hinzugefügt werden, ohne Keys zu generieren. Sollen die Keys jetzt generiert werden?
 
+Cannot\ create\ group=Gruppe kann nicht erstellt werden
+
+Cannot\ create\ group.\ Please\ create\ a\ database\ first.=Gruppe kann nicht erstellt werden. Erstellen sie zuerst eine Datenbank.
```

Comment:
To not confuse real externals shared SQL databases with the internal bib-"database", we changed the wording to library - Bibliothek.
Please change that

Reply:
fixed